gradio/blocks.py

```diff
@@ -250,6 +246,10 @@ def get_config(self):
             "root_url": self.root_url,
         }
 
+    @staticmethod
```

Comment:
If this is the case, then the following would be clearer, no?

```python
@staticmethod
@abstractmethod
```

I'm thinking about custom components. Making it an abstract method would make it clear to devs that the method has to be implemented.

Reply:
Sounds good!